pcmanfm/tabpage.cpp

```diff
@@ -1078,4 +1081,24 @@ bool TabPage::canOpenAdmin() {
     return false;
 }
 
+void TabPage::onSelectItems() {
+    Fm::FileInfoList filesToSelect;
+    for(QString item : filesToSelect_) {
+        Fm::FilePath path = Fm::FilePath::fromPathStr(item.toStdString().c_str());
+        Fm::FileInfoPtr fileInfoPtr = proxyModel_->fileInfoFromPath(path);
+        filesToSelect.push_back(std::move(fileInfoPtr));
+    }
+    folderView_->selectFiles(filesToSelect);
+    // workaround to scroll and select the first item when there's multile files to select
+    if(filesToSelect_.count() > 1) {
```

Comment:
I think you wouldn't need this workaround if you added the contents to `onUiUpdated()` and also set `scrolled` (in the latter) to `true` if `filesToSelect_` wasn't empty. Would you test that?